IAIP/SSCP/SSCPEvents.vb

```diff
@@ -1969,57 +1814,6 @@ Public Class SSCPEvents
             DTPACCPostmarked.Enabled = True
             dtpAccReportingYear.Enabled = True
             txtACCComments.ReadOnly = False
-            If NUPACCSubmittal.Value > 1 Then
-                rdbACCPostmarkYes.Enabled = False
-                rdbACCPostmarkNo.Enabled = False
-                rdbACCROYes.Enabled = False
-                rdbACCRONo.Enabled = False
-                rdbACCCorrectACCYes.Enabled = False
-                rdbACCCorrectACCNo.Enabled = False
-                rdbACCConditionsYes.Enabled = False
-                rdbACCConditionsNo.Enabled = False
-                rdbACCCorrectYes.Enabled = False
-                rdbACCCorrectNo.Enabled = False
-                rdbACCDeviationsReportedYes.Enabled = False
-                rdbACCDeviationsReportedNo.Enabled = False
-                rdbACCPreviouslyUnreportedDeviationsYes.Enabled = False
-                rdbACCPreviouslyUnreportedDeviationsNo.Enabled = False
-                rdbACCEnforcementNeededYes.Enabled = False
-                rdbACCEnforcementNeededNo.Enabled = False
-                rdbACCAllDeviationsReportedYes.Enabled = False
-                rdbACCAllDeviationsReportedNo.Enabled = False
-                rdbACCAllDeviationsReportedUnknown.Enabled = False
-                rdbACCResubmittalRequestedNo.Enabled = False
-                rdbACCResubmittalRequestedUnknown.Enabled = False
-                rdbACCResubmittalRequestedYes.Enabled = False
-                dtpAccReportingYear.Enabled = False
-            Else
-                rdbACCPostmarkYes.Enabled = True
-                rdbACCPostmarkNo.Enabled = True
-                rdbACCROYes.Enabled = True
-                rdbACCRONo.Enabled = True
-                rdbACCCorrectACCYes.Enabled = True
-                rdbACCCorrectACCNo.Enabled = True
-                rdbACCConditionsYes.Enabled = True
-                rdbACCConditionsNo.Enabled = True
-                rdbACCCorrectYes.Enabled = True
-                rdbACCCorrectNo.Enabled = True
-                rdbACCDeviationsReportedYes.Enabled = True
-                rdbACCDeviationsReportedNo.Enabled = True
-                rdbACCPreviouslyUnreportedDeviationsYes.Enabled = True
-                rdbACCPreviouslyUnreportedDeviationsNo.Enabled = True
-                rdbACCEnforcementNeededYes.Enabled = True
-                rdbACCEnforcementNeededNo.Enabled = True
-                rdbACCAllDeviationsReportedYes.Enabled = True
-                rdbACCAllDeviationsReportedNo.Enabled = True
-                rdbACCAllDeviationsReportedUnknown.Enabled = True
-                rdbACCResubmittalRequestedNo.Enabled = True
-                rdbACCResubmittalRequestedUnknown.Enabled = True
-                rdbACCResubmittalRequestedYes.Enabled = True
-                DTPACCPostmarked.Enabled = True
-                chbACCReceivedByAPB.Enabled = True
-                dtpAccReportingYear.Enabled = True
-            End If
 

```

Comment:
Another thing I meant to mention. In this section, the UI gets updated. Previously, the update depended on the submittal value. Now the submittal value will always be "1", but the UI still needs to be updated.

Reply:
Sorry, that code section wasn't quite right. It starts here for the ACC, and also there's a similar one for the Reports:
![image](https://github.com/user-attachments/assets/dcfb14d1-1e1d-4afa-8e39-de3634f4e7df)